### ISSUE TITLE
Typo in MiSTer/Games/NeoGeo/romsets.xml

### ISSUE THREAD

**Issue by johhenrik** [frederic-mahe/Hardware-Target-Game-Database#406]
There's a tiny typo in this file:

MiSTer/Games/NeoGeo/romsets.xml

```
-	<romset name="nam1975" pcm="0" altname="NAM-1975" publisher="SNK" year="1990" vromb_offset="0x200000"/>/>
+	<romset name="nam1975" pcm="0" altname="NAM-1975" publisher="SNK" year="1990" vromb_offset="0x200000"/>
```

Addressed in: https://github.com/MiSTer-devel/NeoGeo_MiSTer/commit/ffaf3ce9145ec3dde18fed4c7f9dd59f7d00f8f0

Probably not worth a new update, but might as well get fixed, because I would assume there might be some ao486 updates coming over time, considering the speed of development of that MiSTer core.

**Comment by frederic-mahe** [comment]
Thanks @johhenrik This is not fixable directly in SMDB files, it must be fixed upstream.

**Comment by johhenrik** [comment]
@frederic-mahe Yeah, I am not sure where to report it though, so I figured it might get noticed through here. Is there a better place to report these types of issues?

**Comment by frederic-mahe** [comment]
No, here is a good place @johhenrik . Is that something that can be fixed upstream, on your side @SmokeMonsterPacks ?

**Comment by johhenrik** [comment]
So, this hasn't been fixed, and I noticed that this package is now handled here, so I would like to just raise this again.

**Comment by tjanas** [comment]
Can you provide checksum information for the expected file?

**Comment by tjanas** [comment]
This is the entry currently:
`ea6c38611acdf19bc42a4458e255e9ebb4c0862aabf7224ca6e6cc7ee4f54745	MiSTer/Games/NeoGeo/romsets.xml	ef79b87c6e1f8c1373d2da43fd2199fa6f55d796	6dafe62d811e0a88f1de915c3bec678a	b5df38c8`

**Comment by johhenrik** [comment]
Latest version of the file (with this fix included) is here: https://github.com/MiSTer-devel/NeoGeo_MiSTer/blob/master/releases/romsets.xml

There are more additions to it I just realized, than just this fix.

Not sure which checksum you need (I apologize for my ignorance), so I added several here.

CRC-32: 4795924C
CRC-64: 4CE5EF2DED6479C6
SHA-256: 032063C8A3B8141AF8E77486A5354F06A1AF08AEDC0B70B66B83701EE1DADE52
SHA-1: 8C5560B476897024E8A5EEC6A024FB5E4EF44F4C

**Comment by johhenrik** [comment]
I realize that maybe it doesn't work to just replace the romset.xml with the new version. So here's the checksum for the changed one, with just the typo fixed.

CRC-32: 1EA568CD
CRC-64: E68C6B5AFF43EBF2
SHA-256: 2A83C129129FF4A2A04C34FD6EEEBFC0082C41887846A011D827613D70B5EE4A
SHA-1: F210DBA1BCC79E994E842702667BEA9DE89AA44A

**Comment by tjanas** [comment]
Can you provide the md5sum too? SMDB entries contain the following:

SHA256, SHA1, MD5SUM, CRC32

**Comment by johhenrik** [comment]
I have edited my comments above and included md5 too.
